frontend/src/scenes/Resources/index.tsx

```diff
@@ -1,83 +1,73 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import styles from './styles.module.scss';
-import { Link } from 'react-router-dom';
-import { Navbar, Nav, Form, FormControl, Table, Image, Button } from 'react-bootstrap';
-import logo from '../../images/retool-logo.png';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faBriefcase, faDatabase } from '@fortawesome/free-solid-svg-icons';
-import { Routes } from '../../common/enums/Routes';
+import { Form, FormControl, Button } from 'react-bootstrap';
+import Loader from '../../components/Loader/index';
+import Header from '../../components/Header/index';
+import TableContainer from './containers/TableContainer/index';
+import { IResource } from '../../common/models/resources/IResource';
+import { fetchResourceRoutine } from './routines/index';
+import { IAppState } from '../../common/models/store/IAppState';
+import { connect } from 'react-redux';
 
-const Resources = () => (
-  <div className={styles['resources-wrp']}>
-    <Navbar className={styles.navigation}>
+interface IProps {
+  resources: IResource[],
+  isLoading: boolean,
+  fetchResources: Function
```

Comment:
this way we will be sure that the function doesn't accept any parametres
```suggestion
  fetchResources: () => void
```